### DISCUSSION TITLE
AirSim Python APIs documentation

### DISCUSSION THREAD

**Discussion by heisenberg898** [microsoft/AirSim#3223] [Support Q&A] [answered]
where can i find AirSim Python APIs documentation 

and i am trying do draw  a polygon and some other shape i cant able to make it correct way 
kindly do the support

More Info
More information on AirSim Python APIs can be found at: https://github.com/Microsoft/AirSim/blob/master/docs/python.md

if i go to the link its page not found

**Comment by GimpelZhang** [comment, marked as answer]
Hi, here are some hints:

[Core APIs](https://microsoft.github.io/AirSim/apis/)

**Reply by heisenberg898** [reply]
thanks

**Comment by rajat2004** [comment]
The reference docs have also been fixed and can be found here - https://microsoft.github.io/AirSim/api_docs/html/

**Comment by jonyMarino** [comment]
Hi @heisenberg898!
Where in the docs is that link?
